src/qt/guiconstants.h

```diff
@@ -7,7 +7,8 @@
 #define BITCOIN_QT_GUICONSTANTS_H
 
 /* Milliseconds between model updates */
-static const int MODEL_UPDATE_DELAY = 250;
+static const int MODEL_UPDATE_DELAY1 = 250;
+static const int MODEL_UPDATE_DELAY2 = 5000;
```

Comment:
For readability, I think it would make more sense to name these constants `MODEL_UPDATE_DELAY_TPS` and `MODEL_UPDATE_DELAY_BAN`, since the original deals with updating the transactions per second and the new one deals with clearing expired bans.